index.html

```diff
@@ -1671,32 +1669,152 @@ <h3>Choosing a definition of 'string'</h3>
-	
-	<p>It is preferable, however, to specify these fields as a {{DOMString}} (or, rarely, a {{USVString}}), since the data encoded into these fields must be serialized from and deserialized into in-memory string representations, such as the [[DOM]] or JavaScript strings or your platform's native Unicode string type.</p>
-</aside>
+    <p>UTF-8 is the preferred [=character encoding=] for wire and document formats on the Web [[ENCODING]] or the Internet in general [[RFC3629]]. When content is encoded in UTF-8, there is rarely a reason to interact with it as a byte sequence. Most Web APIs and interfaces are more concerned with the [=code point=] sequence, since that represents the characters in question, rather than the specific byte values.</p>
+    
+    <p>Sometimes specifications do need to deal with the storage, interpretation, and manipulation of byte values. In particular, many document formats and protocols were defined around the use of 7-bit [[ASCII]] bytes, while allowing the inclusion or interchange of non-ASCII data values via the use of various character or data encoding schemes. Sometimes this is done by designating a [=character encoding form=], such as with the <code>charset</code> parameter of the <code>text</code> media types. Or it might be done by encoding byte values using some special syntax, an example of which would be [=percent encoding=].</p>
+    
+    <p>The prevalence of UTF-8, including as the preferred and default [=character encoding form=], diminishes the need for most specifications to access or manipulate underlying byte values. UTF-8 is designed such that 7-bit ASCII text is also valid UTF-8, which can be important when dealing with encoding or decoding formats based on ASCII.</p>
+    
+    <aside class="example" title="An ASCII based protocol: HTTP 1.1">
+       <p>The HTTP [[RFC9112]] protocol is defined by a series of byte sequences separated by CRLF (carriage return/line feed). Important in the design and specification of HTTP is the fact that the byte values and characters of 7-bit US-ASCII are interchangeable. HTTP does not define or prefer any particular [=character encoding form=] for non-ASCII data; it is agnostic about such values, dealing only with octets (bytes). The specification says:</p>
+    
+       <blockquote>A recipient MUST parse an HTTP message as a sequence of octets in an encoding that is a superset of US-ASCII.</blockquote>
+       
+       <p>An HTTP request encoded in UTF-8 might contain lines that look like this:</p>
+       
+       <pre class="http">
+POST /file-upload HTTP/1.1
+User-Agent: Mozilla/5.0
+Host: www.example.com
+Accept-Language: fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
+Content-Disposition: attachment; filename="Åsa.txt"
```

Comment:
It might be better to use a more abstract example. `Content-Disposition` has a whole UTF-8 scheme as per https://datatracker.ietf.org/doc/html/rfc6266

Reply:
I wanted a realistic example, although these days there aren't many choices in that regard. I suppose I could use `Set-Cookie` or `User-Agent` instead, or perhaps an `X-` header.

Of course, in the real world, there aren't many places where one wants to send bytes down the wire instead of using UTF-8 (and its friend `DOMString` in the API space)